msong98/DynastyBot-Discord
# Add a venv and a basic gitignore

To test

0. Checkout code
1. If required, copy your keyfiles to the project root
2. Load or create a venv
    - `python -m venv venv`
3. Install project requirements
    - `pip install -r requirements.txt`
4. Ensure the bot works as expected
